server/api/dev.js

```diff
@@ -0,0 +1,214 @@
+      /*
+        ------- DELETE OLD DATA -------
+        Ensures developer account is in dev database
+        Create a fake "dev" user
+      */
+      await db.del('votes', { contest_id: 'dev' });
+      await db.del('contest_entries', { contest_id: 'dev' });
+      await db.any("DELETE FROM entries WHERE id LIKE 'dev%';");
+      await db.del('contest_categories', { contest_id: 'dev' });
+
+      /*
+        ------- CONTEST_CATEGORIES -------
+        Create categories for dev contest
+      */
+      const categories = [{ contest_id: 'dev', category: 'cat1' }, { contest_id: 'dev', category: 'cat2' }];
+      await db.insert('contest_categories', categories);
+
+      /*
+        ------- USERS -------
+        Ensures developer account is in dev database
+        Create a fake "dev" user
+      */
+      const users = await db.select('SELECT username FROM users');
+      if (!users.some((user) => user.username === username)) {
+        await db.insert('users', [{ username, contest_reminders: true, moderator: true }]);
+        users.push({ username });
+      }
+      if (!users.some((user) => user.username === 'dev')) {
+        await db.insert('users', [{ username: 'dev', contest_reminders: true, moderator: false }]);
+        users.push({ username: 'dev' });
+      }
+
+      /*
+        ------- ENTRIES -------
+        Create some flags for yourself and dev user
+        Adust as desired
+        NOTE: borrows flags from existing entires but overrides key data
+              accurate url, width, height
+              diversity of images, descriptions, etc
+      */
+      let entryVersions = [
+        { user: 'dev', name: 'Pending example', submission_status: 'pending' },
+        { user: 'dev', name: 'Withdrawn example', submission_status: 'withdrawn' },
+        {
+          user: 'dev', name: 'Approved example', submission_status: 'approved', modified_by: 'VertigoOne',
+        },
+        {
+          user: 'dev', name: 'Rejected example', submission_status: 'rejected', modified_by: 'VertigoOne', rejection_reason: 'Rejected because.',
+        },
+        { user: username, name: 'Pending example', submission_status: 'pending' },
+        { user: username, name: 'Withdrawn example', submission_status: 'withdrawn' },
+        {
+          user: username, name: 'Approved example', submission_status: 'approved', modified_by: 'VertigoOne',
+        },
+        {
+          user: username, name: 'Rejected example', submission_status: 'rejected', modified_by: 'VertigoOne', rejection_reason: 'Rejected because.',
+        },
+      ];
+
+      // add sequential dev# for id (NOTE: zero-indexed, can do i+1 if desired)
+      entryVersions = entryVersions.map((v, i) => ({ ...v, id: `dev${i}`, background_color: BACKGROUND_COLORS[i % 3] }));
+
+      const existingEntries = await db.select(`SELECT width, height, url FROM entries WHERE height > 600 AND POSITION('https://firebasestorage.googleapis.com' IN url) > 0 LIMIT ${entryVersions.length}`);
```

Comment:
Just curious, but why not `AND url LIKE '%firebasestorage%'`?

Reply:
Originally had that, sort of forget why I thought using the entire url felt safer but switched to this